scripts/fuzz_shell.js

```diff
@@ -151,6 +163,18 @@ if (typeof WebAssembly.Tag !== 'undefined') {
   };
 }
 
+// If a second binary will be linked in then set up the imports for
+// placeholders. Any import like  (import "placeholder" "0" (func ..  will be
+// provided by the secondary module, and must be called using an indirection.
+if (secondBinary) {
+  imports['placeholder'] = new Proxy({}, {
+    get(target, prop, receiver) {
+      // Return a function that does an indirect call using the exported table.
+      return (...args) => exports['table'].get(+prop)(...args);
+    }
+  });
+}
```

Comment:
This proxy can just throw an error if it is ever called. Since the secondary module is eagerly loaded, it should be impossible to end up in a placeholder function.

Reply:
Good point!